pkg/tunneldriver/driver.go

```diff
@@ -53,9 +53,9 @@ type TunnelDriverOpts struct {
 }
 
 // New creates and initializes a new TunnelDriver
-func New(logger logr.Logger, opts TunnelDriverOpts) (*TunnelDriver, error) {
+func New(logger logr.Logger, opts TunnelDriverOpts, comments ...string) (*TunnelDriver, error) {
 	connOpts := []ngrok.ConnectOption{
-		ngrok.WithClientInfo("ngrok-ingress-controller", version.GetVersion()),
+		ngrok.WithClientInfo("ngrok-ingress-controller", version.GetVersion(), comments...),
```

Comment:
Since we already have precedent for sending a json-serialized struct as the comment, could we do that instead? Should make it easier to include more complex information should the need ever arise.

We could also tag all of the fields with `omitempty`, and then skip the comment entirely if it serializes to just `{}`.

Or instead of a struct, a `map[string]any` would work if we wanted more flexibility and fewer types to update when we want to send more things.

Reply:
here's a type serialized version: https://github.com/ngrok/kubernetes-ingress-controller/pull/357/commits/311168f82644f9cfb8d55d376e9a273b7f291878

Is this what you meant?